src/sentry/static/sentry/images/logos/logo-victorops.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" id="Layer_3" data-name="Layer 3" viewBox="0 0 55.08 55.32"><defs><style>.cls-1{fill:#fbb040;}</style></defs><circle cx="27.54" cy="27.66" r="27.47" class="cls-1"/><path d="M41.12 28.25L39.32 25l3.9-5.23-11 5-.22-1.5 3.2-4.9-.44-.94 1.47-5.72-5.78 5.7-.33-3.1 1.72-4.77-2.28.73-2-5.72-2 5.72-2.28-.73 1.72 4.8-.33 3.1-5.78-5.7 1.45 5.72-.44.94 3.2 4.9-.22 1.5-11-5 3.88 5.2L14 28.25l8.47 4 .14 1.33-6.35-1.9 3.2 3.8.23 1.2 3.92 1.7.4 1-1.8-.62L26.84 44a62.22 62.22 0 0 1 .3 8.13h.8a62.22 62.22 0 0 1 .32-8.13l4.67-5.24-1.8.62.42-1.06 3.92-1.7.23-1.18 3.2-3.8-6.38 1.9.14-1.34z"/></svg>
```

Comment:
@ceorourke If we are going to add the logo here, could you also update pluginIcon.tsx so Victor Ops has an icon in the integration directory?

Reply:
added!